src/Webform.js

```diff
@@ -1469,6 +1469,11 @@ export default class Webform extends NestedDataComponent {
 
       submission.state = options.state || 'submitted';
 
+      if (this.form._vid) {
```

Comment:
I am pretty sure that this forces the configuration of "Use original form when viewing submissions" and will break "Use current form when viewing submissions".  I think this PR is not good for commit.